New faq (#501)
klaytn/klaytn-docs: docs/misc/faq.md

- [Where can I find complete working samples of fee-delegation?](#fee-delegation-samples)

## What is Klaytn? <a id="what-is-klaytn"></a>

> Klaytn is the blockchain platform with low latency, high TPS and immediate finality. It is the optimal blockchain for building games and realizing the metaverse.

Author: ```suggestion
Klaytn is the blockchain platform with low latency, high TPS and immediate finality. It is the optimal blockchain for building games and realizing the [metaverse](https://docs.klaytn.foundation/klaytn2/metaverse-package).
```

## What is Klaytn 2.0? <a id="what-is-klaytn-2.0"></a>

> Klaytn 2.0 marks Klaytn’s transition to focus on the metaverse, supporting Ethereum Equivalence and the comprehensive Metaverse Package to facilitate blockchain building experience. To better understand Klaytn 2.0, please refer to our Light Paper.

Author: ```suggestion
Klaytn 2.0 marks Klaytn’s transition to focus on the metaverse, supporting Ethereum Equivalence and the comprehensive Metaverse Package to facilitate blockchain building experience. To better understand Klaytn 2.0, please refer to our [Light Paper](https://klaytn.foundation/wp-content/uploads/Lightpaper.pdf).
```

## How does Klaytn support Ethereum equivalence? <a id="how-ethereum-equivalence"></a>
Klaytn is EVM compatible, and supports all Ethereum London EVM features. Our `eth` namespace has been amended to be compatible with Ethereum’s. Ethereum toolings can be used seamlessly, and migration of Ethereum dApps have also become possible with minimal effort. Transaction types and fields are also equivalent to Ethereum’s.

## What is Klaytn’s gas policy? <a id="klaytn-gas-policy"></a>

> Klaytn transitioned from a fixed gas fee policy to adopt a dynamic gas fee mechanism in order to combat arbitrage bots.

Reviewer: Also could be better to add a link to find more detailed description of this.

Reviewer 2: To add to this, there are some links attached to a number of sections in the FAQs [google docs](https://docs.google.com/document/d/1Mj-_temp9DHs-LaDZAqKsftTw8gjGudKy_Yn7faI49M/edit), that's not reflecting here.

> Klaytn transitioned from a fixed gas fee policy to adopt a dynamic gas fee mechanism in order to combat arbitrage bots.

Author: ```suggestion
Klaytn transitioned from a fixed gas fee policy to adopt a dynamic gas fee mechanism in order to combat arbitrage bots. Please refer to this [article](https://medium.com/klaytn/dynamic-gas-fee-pricing-mechanism-1dac83d2689) for more information.
```

## What is special about Klaytn’s account structure? <a id="klaytn-account-structure"></a>

> To provide maximum convenience for dApp developers, Klaytn has devised a way to decouple private keys from addresses. As a result, you can easily implement multisig, whereby you create multiple private keys for a single account, with each key having different weights. Each key can be assigned with different roles as well.

Author: ```suggestion
To provide maximum convenience for dApp developers, Klaytn has devised a way to [decouple private keys from addresses](https://klaytn-tech.medium.com/klaytn-usability-improvement-series-1-separating-keys-and-addresses-dd5e367a0744). As a result, you can easily implement [multisig](https://medium.com/klaytn/klaytn-usability-improvement-series-2-introducing-multisig-on-the-platform-level-85141893db01), whereby you create multiple private keys for a single account, with each key having different weights. Each key can be assigned with [different roles](https://medium.com/klaytn/klaytn-usability-improvement-series-4-supporting-role-based-keys-on-the-platform-level-e2c912672b7b) as well.
```

## Where can I start dApp development with Klaytn? <a id="dapp-development"></a>

> Whether you are migrating from Ethereum, or building on Klaytn from scratch, we support all the necessary tools and infrastructure. You can test your smart contracts on Remix IDE using Klaytn Plugin or connect to MetaMask wallet and Kaikas. Klaytn’s sdk `caver` is available as caver-js You can refer to our tutorials to try building a dApp on Klaytn.

Author: ```suggestion
Whether you are migrating from Ethereum, or building on Klaytn from scratch, we support all the necessary tools and infrastructure. You can test your smart contracts on [Remix IDE](https://docs.klaytn.foundation/dapp/tutorials/connecting-remix) using Klaytn Plugin or connect to [MetaMask](https://docs.klaytn.foundation/dapp/tutorials/connecting-metamask) wallet and [Kaikas](https://chrome.google.com/webstore/detail/kaikas/jblndlipeogpafnldhgmapagcccfchpi). Klaytn’s sdk `caver` is available as [caver-js](https://github.com/klaytn/caver-js) You can refer to our [tutorials](https://docs.klaytn.foundation/dapp/tutorials) to try building a dApp on Klaytn.
```

## Is Klaytn Open Source? <a id="is-klaytn-open-source"></a>

> Klaytn is most certainly open source! Take a look at our Github Organisation and you can start contributing to our Klaytn Documentation. Read more about our open-source policies here.

Author: ```suggestion
Klaytn is most certainly open source! Take a look at our [Github Organisation](https://github.com/klaytn) and you can start [contributing](https://github.com/klaytn/klaytn-docs/blob/master/CONTRIBUTING.md) to our Klaytn Documentation. Read more about our open-source policies [here](https://docs.klaytn.foundation/misc/opensource).
```

## How can I initially fund my account? <a id="fund-my-acconut"></a>

> You may purchase KLAY on the exchange. The list of available exchanges can be found in Coinmarketcap or Coingecko.

Author: ```suggestion
You may purchase KLAY on the exchange. The list of available exchanges can be found here:
[Coinmarketcap](https://coinmarketcap.com/currencies/klaytn/markets/) or [Coingecko](https://www.coingecko.com/en/coins/klay#markets)
```

- [Where can I find complete working samples of fee-delegation?](#fee-delegation-samples)

## What is Klaytn? <a id="what-is-klaytn"></a>

> Klaytn is the blockchain platform with low latency, high TPS and immediate finality. It is the optimal blockchain for building games and realizing the [metaverse](https://docs.klaytn.foundation/klaytn2/metaverse-package).

Reviewer 3: ```suggestion
Klaytn is the Layer 1 blockchain platform with low latency, high TPS and immediate finality. It is the optimal blockchain for building games and realizing the [metaverse](https://docs.klaytn.foundation/klaytn2/metaverse-package).
```

## Is Klaytn Open Source? <a id="is-klaytn-open-source"></a>
Klaytn is most certainly open source! Take a look at our [Github Organisation](https://github.com/klaytn) and you can start [contributing](https://github.com/klaytn/klaytn-docs/blob/master/CONTRIBUTING.md) to our Klaytn Documentation. Read more about our open-source policies [here](https://docs.klaytn.foundation/misc/opensource).

## How can I initially fund my account? <a id="fund-my-acconut"></a>
You may purchase KLAY on the exchange. The list of available exchanges can be found here:
[Coinmarketcap](https://coinmarketcap.com/currencies/klaytn/markets/) or [Coingecko](https://www.coingecko.com/en/coins/klay#markets)

## Any Klaytn’s Public Node Providers for testing and development? <a id="node-providers"></a>

> Refer to this list for Klaytn’s Public Node Providers and the network domains.

Author: ```suggestion
Refer to this [list](https://docs.klaytn.foundation/dapp/json-rpc/public-en#public-json-rpc-endpoint-providers) for Klaytn’s Public Node Providers and the network domains.
```

## Are there faucets to get test KLAY? <a id="are-there-faucets"></a>
You can get test KLAY for development and testing purposes here:

> https://baobab.wallet.klaytn.foundation/faucet

Author: ```suggestion
[Klay Faucet](https://baobab.wallet.klaytn.foundation/faucet)
[AllThatNode Faucet](https://www.allthatnode.com/faucet/klaytn.dsrv)
```

> https://www.allthatnode.com/faucet/klaytn.dsrv

Author: ```suggestion
```

## How do I check for Public RPC endpoint status? <a id="rpc-endpoint-status"></a>

> Since we cannot guarantee uptime and stability of the endpoints, you can always check for node provider status here.

Author: ```suggestion
Since we cannot guarantee uptime and stability of the endpoints, you can always check for node provider status [here](https://www.allthatnode.com/klaytn.dsrv).
```

## Which wallets support Klaytn? <a id="which-wallets"></a>

> Klaytn is supported by the cold wallet D’cent, as well as a host of hot wallets like Kaikas, MetaMask and more. Please refer to the list here.

Author: ```suggestion
Klaytn is supported by the cold wallet D’cent, as well as a host of hot wallets like Kaikas, MetaMask and more. Please refer to the list [here](http://klaytn.foundation/ecosystem).
```

> No official browser wallets at the moment.

Author: ```suggestion
Klaytn's web brower extension wallet [Kaikas](https://chrome.google.com/webstore/detail/kaikas/jblndlipeogpafnldhgmapagcccfchpi?hl=en). Kaikas is a non-custodial wallet with which you can make KLAY transactions and create accounts. 
```

> Kaikas, a browser extension wallet of Klaytn will be released in 1H 2020.

Reviewer 4: Hi @terrikwak , Should we update this as well about the extension ?

Author: @praveen-klaytn Oops! Thanks so much for pointing it out!

> Kaikas, a browser extension wallet of Klaytn will be released in 1H 2020.

Author: ```suggestion
```
